lib/ngtsc/component.symbol.ts

```diff
@@ -16,16 +17,23 @@ export class ComponentSymbol extends Symbol<'Component'> {
     }
   }
 
-  get deps() {
-    return this.metadata.deps;
+  protected get deps() {
```

Comment:
I feel this one should remain public as there is no other "easy" way to access it as the analysis interfaces are not consistent across every annotation.

Reply:
Ideally we should resolve all the deps as `Injectable`s. I'll talk to Alex when we get back in the office. My guess is there wouldn't be an easy way because currently DI happens at runtime, so the compiler doesn't has much to do with it.

Reply:
Agreed, I spend much time on that but couldn't find the right solution :/.